src/rendering/batcher/shared/Batcher.ts

```diff
@@ -73,30 +77,54 @@ function returnBatchToPool(batch: Batch)
 {
     batchPool[batchPoolIndex++] = batch;
 }
-export interface BatchableObject
+
+export interface BatchableElement
```

Comment:
All these interfaces should now be documented as we are turning this into a more public api

Reply:
agree!